tripal_chado/includes/TripalImporter/FASTAImporter.inc

```diff
@@ -651,7 +669,7 @@ class FASTAImporter extends TripalImporter {
       ], $values);
       if (count($results) > 1) {
         $this->logMessage("Multiple features exist with the name '!name' of type '!type' for the organism.  skipping",
-          ['!name' => $name, '!type' => $type], TRIPAL_ERROR);
```

Comment:
`$type` is not an initialized variable.  I think the cvterm name is the right thing here (and below).

Reply:
Yes, I agree.